Picturesbooks.csv

```diff
@@ -0,0 +1,11 @@
+id,cat,name,price,inStock,author,series_t,sequence_i,genre_s
```

Comment:
What file is this?
This was added in previous PR too

Reply:
I have done all changes. and push to my branch